tests/unit/test_system.py

```diff
@@ -73,24 +88,24 @@ class TestSYSCtlChecks(SystemTestsBase):
 
     def test_sysctl_checks(self):
         expected = {'juju-charm-sysctl-mismatch': {
-                        'kernel.pid_max': {
-                            'conf': '50-ceph-osd-charm.conf',
-                            'actual': '4194304',
-                            'expected': '2097152'}}}
+            'kernel.pid_max': {
+                'conf': '50-ceph-osd-charm.conf',
+                'actual': '4194304',
+                'expected': '2097152'}}}
```

Comment:
i assume this is your python editor making autocorrections? is it possible to avoid it?

Reply:
I ran the code formatter to sort out the pylint-flake8 issues, guess it formatted these too. reverted back to original